randovania/gui/ui_files/preset_am2r_patches.ui

```diff
@@ -213,6 +213,27 @@
              <property name="text">
               <string>Ensures all Save Station doors are normal (blue) doors, even with door lock rando enabled.</string>
              </property>
+             <property name="wordWrap">
+              <bool>true</bool>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QCheckBox" name="force_blue_labs_check">
+             <property name="text">
+              <string>Unlock Genetics Laboratory Doors</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="force_blue_labs_check_2">
+             <property name="text">
+              <string>Ensures that the doors in the later parts of Genetics the are normal (blue) doors, even with door lock rando enabled.
```

Comment:
> [...] Genetics `the` are [...]
Remove `the` ?

Reply:
multiple words are missing there, absolutely no idea how that happened